packages/jest-runtime/src/__tests__/transform-test.js

```diff
@@ -84,14 +84,14 @@ const getCachePath = (fs, config) => {
   return null;
 };
 
+let ScriptTransformer;
 let config;
 let fs;
 let mockFs;
 let object;
-let transform;
 let vm;
 
-describe('transform', () => {
+describe('ScriptTransformer', () => {
```

Comment:
Let's also rename the test file (and corresponding snapshot) to `ScriptTransformer-test.js` for consistency.